### DISCUSSION TITLE
Vulkan Output

### DISCUSSION THREAD

**Discussion by gamelounge78** [obsproject/obs-studio#5057] [Development Questions]
Would there be any considerations in adding Vulkan as a new output?  I'm not sure if anyone asked even though I searched the topics for this.

**Comment by Mister36MN** [comment]
Not really sure what you mean by "output", Vulkan is a Graphics API, as in game engines use it to render graphics. OBS can capture Vulkan games as of OBS 25 using game capture.

**Comment by jp9000** [comment]
As Mister36MN said, what do you mean by output?
